### ISSUE TITLE
Enhance the search dialog: correctly support searching in a split window

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#800]
**Reported by Stanzilla on 29 May 2010 19:53:16 UTC**
## I'm lazy so I'll just copy the irc log when I detailed what my problem is. 

```
21:35:11 ∙ Stanzilla: when you split the window and hit ctrl +f for search, the window appears in the wrong part http://yfrog.com/jpkvircbug2p
21:35:14 ∙ Stanzilla: can someone confirm?
21:37:02 ∙ ambossarm: the appearance in the "non message section" isn't that bad, but that is searches only this area is bad
21:37:44 ∙ ambossarm: and it if the non-message-part is so little that it is out of sight, you cannot find the search box
21:39:26 ∙ Stanzilla: yeah and how do i get the search window for the lower part?
21:44:03 ∙ Pragma: you click on the loupe button in the lower part
21:45:07 ∙ Stanzilla: i meant via a shortcut
21:45:14 ∙ Pragma: you can't
21:45:21 ∙ Pragma: :D
21:45:28 ∙ Stanzilla: imo ctrl f should open a search window for both parts
21:45:43 ∙ Stanzilla: that ignores the size of those for positioning and just sits in the center
21:46:10 ∙ ambossarm: both parts? I think if I search something I search text in a message
21:46:17 ∙ ambossarm: ok I never used that feature
21:46:46 ∙ Stanzilla: or when someone joined..hm
21:47:19 ∙ Stanzilla: maybe just add checkboxes where you want to search in
```

---

Migrated from: https://svn.kvirc.de/kvirc/ticket/800

**Comment by DarthGandalf** [comment]
**Modified by ctrlaltca on 13 Jun 2010 16:45:56**
Summary: Problem with the search dialog and split view → Enhance the search dialog: correctly support searching in a split window
Type: Bug → Feature request
Version: 4.0rc3 Insomnia → 4.0 Insomnia

**Comment by DarthGandalf** [comment]
**Modified by HelLViS69 on 27 Jun 2010 15:12:17**
Version: 4.0 Insomnia → 4.1 Equilibrium

**Comment by Stanzilla** [comment]
I think this is still broken.

**Comment by un1versal** [comment]
its broken hence confirmed status but marked as bug as well for now.

**Comment by un1versal** [comment]
Well, Ill make this a feature request again and hope some kind soul looks at these reports some time.

**Comment by ghost** [comment]
This is related to #1319

**Comment by IceN9ne** [comment]
Fixed in #2132
